public/app/features/alerting/unified/components/rule-list/RuleList.v1.tsx

```diff
@@ -105,28 +103,24 @@ const RuleList = withErrorBoundary(
       // We don't want to show the Loading... indicator for the whole page.
       // We show separate indicators for Grafana-managed and Cloud rules
       <AlertingPageWrapper navId="alert-list" isLoading={false} actions={hasAlertRulesCreated && <CreateAlertButton />}>
-        <RuleListErrors />
-        <RulesFilter onFilterCleared={onFilterCleared} />
-        {hasAlertRulesCreated && (
-          <>
-            <div className={styles.break} />
-            <div className={styles.buttonsContainer}>
-              <div className={styles.statsContainer}>
-                {view === 'groups' && hasActiveFilters && (
-                  <Button
-                    className={styles.expandAllButton}
-                    icon={expandAll ? 'angle-double-up' : 'angle-double-down'}
-                    variant="secondary"
-                    onClick={() => setExpandAll(!expandAll)}
-                  >
-                    {expandAll ? 'Collapse all' : 'Expand all'}
-                  </Button>
-                )}
-                <RuleStats namespaces={filteredNamespaces} />
-              </div>
-            </div>
-          </>
-        )}
+        <Stack direction="column">
```

Comment:
Why do we need this stack?

Reply:
Oh probably a left-over from refactoring something locally? We certainly don't need it here – I'll revert this bit.